pom.xml

```diff
@@ -3,6 +3,7 @@
  DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
  Copyright 2011-2015 ForgeRock AS.
+ Portions Copyright © 2017 Wren Security. All rights reserved.
```

Comment:
I think we should pick standard format and stick with it. In Wren:DS I saw `Portions Copyright 2017 Wren Security.` being used.

Reply:
@siepkes can you explain what's different? I'm not sure I follow what you mean. it may have been the case that I was being consistent with the format in the same area of the code.

Reply:
The line in this file also has `All rights reserved.` appended and a copyright sign (`©`)in it . The other files just read `Portions Copyright 2017 Wren Security.`

Yeah it's nit picking, I know ;-) But I think we should be consistent with the copyright header. IANAL so I have not preference as to what it reads. `Portions Copyright 2017 Wren Security.`  should be fine I guess for example.

Reply:
ok, going forward we can do that. previously it was more of a "when in Rome" style -- whatever the header in that area recommended is what I'd use. the FR copyright plugin (now the Wren copyright plugin) will accept a lot of different styles, so I didn't really think much about it. But, yes -- `Portions Copyright © 2017 Wren Security. All rights reserved.` will be the preferred style. If you see any that aren't following that, please call 'em out.